src/models/test.ts

```diff
@@ -25,13 +33,22 @@ export type Test = {
   executorArgs?: string[];
   executionRequest: ExecutionRequest;
   testIcon?: string;
+  status?: {
+    latestExecution?: LatestExecution;
+  };
+  source: string;
 };
 
 export type TestWithExecution = {
   test: Test;
   latestExecution?: Execution;
 };
 
+export type TestWithExecutionRedux = {
+  dataItem: Test;
+  latestExecution?: Execution;
+};
```

Comment:
Is there a reason why we have it stored differently in Redux? Maybe it will be worth creating a follow-up task to align them